models/igmp_if_p.go

```diff
@@ -0,0 +1,81 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/container"
+)
+
+const (
+	RnIgmpIfP        = "igmpIfP"
+	DnIgmpIfP        = "uni/tn-%s/out-%s/lnodep-%s/lifp-%s/igmpIfP"
+	ParentDnIgmpIfP  = "uni/tn-%s/out-%s/lnodep-%s/lifp-%s"
+	IgmpIfPClassName = "igmpIfP"
+)
+
+type IGMPInterfaceProfile struct {
+	BaseAttributes
+	IGMPInterfaceProfileAttributes
+}
+
+type IGMPInterfaceProfileAttributes struct {
+	Annotation string `json:",omitempty"`
+	Name       string `json:",omitempty"`
+	NameAlias  string `json:",omitempty"`
+}
+
+func NewIGMPInterfaceProfile(igmpIfPRn, parentDn, description string, igmpIfPAttr IGMPInterfaceProfileAttributes) *IGMPInterfaceProfile {
+	dn := fmt.Sprintf("%s/%s", parentDn, igmpIfPRn)
```

Comment:
igmpIfPRn is static string without identifiers, why do we need to pass it in as variable?

Reply:
To maintain consistency and to not hardcode static values

Reply:
I agree with Akini on this. We should fix it and if needed open an issue against autogen-v2

Reply:
ok